backend/tests/views/annotation_task.py

```diff
@@ -18,14 +18,14 @@ def test_campaign_list_unauthenticated(self):
         response = self.client.get(url)
         self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
 
-    def test_campaign_user_list_unauthenticated(self):
-        """AnnotationTask view 'campaign_user_list' returns 401 if no user is authenticated"""
-        url = reverse(
-            "annotation-task-campaign-user-list",
-            kwargs={"campaign_id": 1, "user_id": 1},
-        )
-        response = self.client.get(url)
-        self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+    # def test_campaign_user_list_unauthenticated(self):
+    #     """AnnotationTask view 'campaign_user_list' returns 401 if no user is authenticated"""
+    #     url = reverse(
+    #         "annotation-task-campaign-user-list",
+    #         kwargs={"campaign_id": 1, "user_id": 1},
+    #     )
+    #     response = self.client.get(url)
+    #     self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
```

Comment:
Same remark about quoted tests 🙂 As I said earlier, don't hesitate to create a ticket on Jira if you want to do it later!

Reply:
I removed them, it was about a route that has been removed